src/app/client/src/app/modules/learn/components/course-consumption/course-player/course-player.component.ts

```diff
@@ -124,7 +125,8 @@ export class CoursePlayerComponent implements OnInit, OnDestroy {
     private courseConsumptionService: CourseConsumptionService, windowScrollService: WindowScrollService,
     router: Router, public navigationHelperService: NavigationHelperService, private userService: UserService,
     private toasterService: ToasterService, private resourceService: ResourceService, public breadcrumbsService: BreadcrumbsService,
-    private cdr: ChangeDetectorRef, public courseBatchService: CourseBatchService) {
+    private cdr: ChangeDetectorRef, public courseBatchService: CourseBatchService, public externalUrlpreviewService:
+    ExternalUrlpreviewService) {
```

Comment:
why is the variable name is not in lowerCamelCase

Reply:
class name rule is to start with uppercase